eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Parser/Page.php

```diff
@@ -78,8 +79,16 @@ public function preMap(array $config, ContextualizerInterface $contextualizer)
 
         // filters blocks and layouts for each siteaccess to keep only
         // the enabled ones for this sa
+        $configResolver = new ConfigResolver(
+            $container->getParameter('ezpublish.siteaccess.groups_by_siteaccess'),
+            $container->getParameter('ezpublish.config.default_scope')
+        );
+        $configResolver->setContainer($container);
+
         foreach ($config['siteaccess']['list'] as $sa) {
-            $ezpageSettings = $container->getParameter("ezsettings.$sa.ezpage");
+            $siteAccess = new SiteAccess($sa);
+            $configResolver->setSiteAccess($siteAccess);
+            $ezpageSettings = $configResolver->getParameter('ezpage');
```

Comment:
`$container->getParameter("ezsettings.$sa.ezpage")` caused errors because only siteaccesses with ezpage settings will be available now.

The config resolver is necessary to fallback to the default scope.